### ISSUE TITLE
init.vim not loading, no error with "vscode-neovim.neovimInitPath": "nonsense"

### ISSUE THREAD

**Issue by David-Else** [vscode-neovim/vscode-neovim#234]
I have an `init.vim` that is loaded by neo vim OK. Is this config the right way to get the functionality I want in the extension?
- map `;` to `:`
- stop the search highlights
- use space as a leader key
- turn on spelling
- toggle zen mode on and off
```
nnoremap ; :
let mapleader="\<SPACE>"
set nohlsearch
set spell spelllang=en_us

if exists('g:vscode')
    " VSCode extension
    nnoremap <silent> <leader>tz <Cmd>call VSCodeCall('workbench.action.toggleZenMode')<CR>
else
   :inoremap jj <Esc>
endif
```
All the functionality listed works fine in neo vim 0.43, but in the extension nothing happens. My settings are:
```
  "vscode-neovim.neovimInitPath": "/home/david/.config/nvim",
  "vscode-neovim.neovimPath": "/home/david/bin/nvim",
```
It finds the vim executable fine. It is the linux appimage. If i change the `vscode-neovim.neovimInitPath` to a something that does not exist it gives NO error and just continues, I don't know if it is loading or if there is something non compatible with my config? This fails silently:
```
"vscode-neovim.neovimInitPath": "nonsense",
```
THANKS!

**Comment by jensbodal** [comment]
Try:

```
"vscode-neovim.neovimInitPath": "/home/david/.config/nvim/init.vim",
```

Also make sure `which nvim` returns `/home/david/bin/nvim`

**Comment by David-Else** [comment]
@jensbodal  Thank, it now works! I would still count that it is a bug as there should be a user error when the file is not found.

I looked at the commits and see there is a patch that maybe is related to this https://github.com/asvetliakov/vscode-neovim/pull/157 , but there has been no new version released with it in. Do you happen to know the maintainer, he seems to have disappeared for 3 months :(

I am finding this to be the best and most essential extension since I installed it, I really hope he returns soon and releases a new version with all the patches that are waiting!

**Comment by jensbodal** [comment]
I don't, I just came across this plugin after struggling with the slowness of VSCodeVim for too long. 

The maintainer mentioned [here](https://github.com/asvetliakov/vscode-neovim/issues/209#issuecomment-609573058) that he's been sorting out life problems, so hopefully everything is going ok.  

So I don't know when the next release will be but the latest version of the source code fixes the issue in #209 so I just compiled it myself and added the extension manually.  Details are in that PR if you want to do so yourself.
